corehq/apps/zapier/api/v0_5.py

```diff
@@ -162,3 +162,54 @@ def obj_get_list(self, bundle, **kwargs):
 
     class Meta(BaseZapierCustomFieldResource.Meta):
         resource_name = 'custom_fields_case'
+
+
+class ZapierCustomActionFieldCaseResource(BaseZapierCustomFieldResource):
+    """
+        Generates custom action fields for updating or creating a case.  When creating a case, users must provide
+        a new case name, as well as optionally providing values for the case properties.  When updating a case,
+        users must provide a case id, as well as optionally providing new values for the case properties.
+    """
+    def obj_get_list(self, bundle, **kwargs):
+        custom_fields = []
+        domain = bundle.request.GET.get('domain')
+        case_type = bundle.request.GET.get('case_type')
+        create = bundle.request.GET.get('create')
+
+        if create == "True":
+            custom_fields.append(CustomField(
+                dict(
+                    type='unicode',
+                    key='case_name',
+                    label='Case name'
+                )
+            ))
+        else:
+            custom_fields.append(CustomField(
+                dict(
+                    type='unicode',
+                    key='case_id',
+                    label='Case ID'
+                )
+            ))
+            custom_fields.append(CustomField(
+                dict(
```

Comment:
this gets added in both cases, so can you just pull it outside of the if statement?